internal/gateway/proxy.go

```
@@ -3301,6 +3301,13 @@ func (p *GuardrailProxy) enqueueBlockNotification(verdict *ScanVerdict, directio
 	if verdict.Findings != nil {
 		findings = len(verdict.Findings)
 	}
+	// Cloud-controlled per-inspection redaction (phase 1, all-sinks
+	// scope): in managed_enterprise the inspect response's
+	// is_redaction_enabled directive overrides local privacy config for
+	// this verdict's reason. Outside managed_enterprise this resolves to
+	// SinkPolicyDefault, so the existing ForSinkReason behavior is
+	// preserved.
+	policy := sinkPolicyFor(nil, verdict.RedactionEnabled)
```

golangci-lint (2.12.2)

[error] SA1012: do not pass a nil Context, even if a function permits it; pass context.TODO if you are unsure about which Context to use

(staticcheck)